app/models/wpcc/contributions_calendar.rb

```diff
@@ -0,0 +1,56 @@
+module Wpcc
+  class ContributionsCalendar
+    attr_reader :eligible_salary,
+                :salary_frequency,
+                :employee_percent,
+                :employer_percent
+
+    PERIODS_FILE = Wpcc::Engine.root.join('config', 'contribution_periods.yml')
+    PERIODS = YAML.load_file(PERIODS_FILE)
+
+    def initialize(eligible_salary:,
+                   employee_percent:,
+                   employer_percent:,
+                   salary_frequency:)
+
+      @eligible_salary = eligible_salary
+      @employee_percent = employee_percent
+      @employer_percent = employer_percent
+      @salary_frequency = salary_frequency
+    end
+
+    def schedule
```

Comment:
@tomas-stefano @margOnline is it possible to have public methods that allows us to pick out _named_ periods. We have named the periods as _current_, _next_ and _onwards_. So that we can do things like `@pension_contribution.current`, `@pension_contribution.next` and `@pension_contribution.onwards`.

i.e. 
```
def current
  schedule.first
end
.
.
.
```

```
@pension_contribution.current =>
=> [#<Wpcc::PeriodContribution:0x007fd9450e1278
  @employee_contribution=33.33,
  @employer_contribution=16.67,
  @tax_relief=6.67,
  @total_contributions=50.0>
```

```
@pension_contribution = Wpcc::ContributionsCalendar.new(eligible_salary: 20000, employer_percent: 1, employee_percent: 2, salary_frequency: 12)

=> #<Wpcc::ContributionsCalendar:0x007fd93dfb0590
 @eligible_salary=20000,
 @employee_percent=2,
 @employer_percent=1,
 @salary_frequency=12>
[13] pry(main)> cc.schedule
=> [#<Wpcc::PeriodContribution:0x007fd9450e1278
  @employee_contribution=33.33,
  @employer_contribution=16.67,
  @tax_relief=6.67,
  @total_contributions=50.0>,
 #<Wpcc::PeriodContribution:0x007fd9450e1048
  @employee_contribution=50.0,
  @employer_contribution=33.33,
  @tax_relief=10.0,
  @total_contributions=83.33>,
 #<Wpcc::PeriodContribution:0x007fd9450e0e18
  @employee_contribution=83.33,
  @employer_contribution=66.67,
  @tax_relief=16.67,
  @total_contributions=150.0>]
```